qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -501,7 +501,7 @@ def new_carr_purcell_meiboom_gill_sequence(
     )
 
 
-def new_uhrig_single_axis_sequence(
+def new_single_axis_uhrig_sequence(
```

Comment:
Google tells me that UDD is a thing -- is that the same as this, or is the "single axis" special somehow?

Reply:
That's a good question. I wasn't sure about what single-axis here means in the first place. Looking at the internal document and [wiki](https://docs.q-ctrl.com/wiki/control-library), it is not documented anyway. @perjlieb  any idea about why we had this `single-axis` stated explicitly before?

Reply:
@michaelhush might have a better idea here.

Reply:
The single-axis refers to one of the x, y and z axes, which can be generalized to a fixed control axis (fixed phase in time). The sequence reduces the noise along this single-axis. Probably @michaelhush and @HarrisonBall can tell you more about this.

Reply:
Thanks for the explanation. Would UDD by default mean a single-axis sequence? If so, can we just remove the term single axis in this case (and for others)? As spin echo, CP, and CPMG are also single-axis sequences in this sense, and we don't use single-axis for them, IIUC.

Reply:
https://royalsocietypublishing.org/doi/pdf/10.1098/rsta.2011.0355
From this paper, the impression I have now is a multi-axis suquence means a combination of two or more types of  rotations along differnt axis. As UDD means Y rotations applied at specific time offsets, I think it should be safe to leave the `single-axis` out (and also for `periodic` and `walsh`).

Reply:
I think it is safe to leave `single-axis` out as well. I consulted the `numerical-package` to find out where `single-axis`  is used and it basically is a flag whether pulses have contributions to one ore more major axes (x,y, and z).

Reply:
SG, I'll remove `single-axis` from function names.